getnyumba/node-api
# #166730609 Setup node/express project.

**What does this PR do?**
Sets up an empty node and express project for GetNyumba.

**Description of the task to be completed**
The project should be set up with a Node/Express backend environment and run a MongoDB database with mongoose.

**How should this be manually tested?**

- Clone the repository `https:github.com/getnyumba/node-api.git`
- Checkout to the `ch-setup-node-project-166730609`
- Run the app using `node index.js`
- The terminal should display the text `GetNyumba running on port: 8000`
- Run http://localhost:8000/ in a browser which should display `Hello house hunters`
